src/locales/en/translation.json

```diff
@@ -114,7 +114,7 @@
   "createWallet": {
     "confirmSaved": "I saved my keyphrase",
     "doNotShare": "Never share your keyphrase, anyone with your keyphrase can access your wallet and your tokens.",
-    "instruction": "Save your keyphrase <1>in the right order</1> in a secure location, you will need it to open your wallet.",
+    "instruction": "Save your keyphrase <strong>in the right order</strong> in a secure location, you will need it\n                to open your wallet.",
```

Comment:
Why you need those weird whitespaces and why only in EN?

Reply:
Oh, good catch. Must have come from `yarn lint` and `yarn extract-messages`
```jsx
                defaults="Save your keyphrase <strong>in the right order</strong> in a secure location, you will need it
                to open your wallet."
```